docs/source/cross_validation.rst

```diff
@@ -24,6 +24,47 @@ The interface for splitting Dask arrays is the same as scikit-learn's version.
 
    X_train.compute()[:3]
 
+Here is another illustration of performing k-fold cross validation purely in Dask. Here a link to gather more information on k-fold cross validation :func:`https://ml.dask.org/modules/generated/dask_ml.model_selection.KFold.html`:
```

Comment:
All of this should go at the bottom of the document (the next two paragraphs are discussing train_test_split still)

Reply:
incorporated your suggestions, moved k-fold to the bottom of the document